### ISSUE TITLE
Dịch hình chương 13 - Phần 1

### ISSUE THREAD

**Issue by rootonchair** [aivivn/d2l-vn#2245]
Các hình cần dịch:
* [finetune.svg](https://github.com/aivivn/d2l-vn/blob/master/img/finetune.svg)
* [iou.svg](https://github.com/aivivn/d2l-vn/blob/master/img/iou.svg)
* [anchor-label.svg](https://github.com/aivivn/d2l-vn/blob/master/img/anchor-label.svg)
* [ssd.svg](https://github.com/aivivn/d2l-vn/blob/master/img/ssd.svg)

Xem hướng dẫn dịch hình ảnh **[tại đây](https://github.com/aivivn/d2l-vn/blob/master/img/README.md)**.

**Comment by cuongvng** [comment]
mình nhận 3 phần này nha.

**Comment by rootonchair** [comment]
Mình vẫn chưa làm hướng dẫn edit hình. @cuongvng biết làm bước đó không nhỉ?

**Comment by cuongvng** [comment]
Mình nghĩ chủ yếu là làm quen với Inkscape thôi, quen rồi thì mấy cái xóa rồi chèn text cũng đơn giản. 
Mình không thu xếp thời gian viết tỉ mỉ được, sorry nha. Mà anh em làm engineering chắc vọc google cũng nhanh thôi.

**Comment by rootonchair** [comment]
> Mình không thu xếp thời gian viết tỉ mỉ được, sorry nha. Mà anh em làm engineering chắc vọc google cũng nhanh thôi.
Sorry, mình nói dễ hiểu nhầm. Mình muốn chắc rằng bạn không ngại việc edit hình thôi. Phần tutorial mình sẽ làm. 😄

**Comment by cuongvng** [comment]
à sorry mình hiểu nhầm =)), trước mình có edit hình rồi mà #1682

**Comment by rootonchair** [comment]
Nice! Mình không nhớ...

**Comment by cuongvng** [comment]
review phần dịch text giúp mình nhé.

|Tiếng Anh|Tiếng Việt|
| :---:   | :---:    |
| Pre-train | Tiền huấn luyện |
| Source model | Mô hình nguồn |
| Target model | Mô hình đích |
| Source data | Dữ liệu nguồn |
| Target data | Dữ liệu đích |
| Train from scratch  | Huấn luyện từ đầu |
| Fine-tune | Tinh chỉnh |
| Random initialization | Khởi tạo ngẫu nhiên |
| Copy | Sao chép |
| Output layer | Tầng đầu ra |
| Intersection | Phần giao |
| Union | Phần hợp |
| IoU (Intersection over Union) | Giao trên hợp |
| Ground-truth bounding box index | Chỉ số khung chứa nhãn gốc |
| Anchor box index | Chỉ số khung neo |
| Category prediction | Dự đoán nhãn |
| Bounding box prediction | Dự đoán khung chứa |
| Multiscale feature | Đặc trưng đa tỉ lệ |
| Base network | Mạng cơ sở |

**Comment by lkhphuc** [comment]
Category prediction -> Dự đoán hạng mục

**Comment by rootonchair** [comment]
IoU (Intersection over Union) -> Giao trên hợp (IoU)

Bạn có thể đưa vào hình nhé

**Comment by cuongvng** [comment]
à `pre-train` là `huấn luyện sẵn` thì hợp hơn nhỉ?

**Comment by rootonchair** [comment]
Lạ nhỉ? Nếu vậy phải là `pre-trained` mới đúng

**Comment by lkhphuc** [comment]
Mình thấy `tiền huấn luyện` cũng được, không chuẩn như `đã qua huấn luyện` nhưng vẫn đủ hiểu (theo mình) và dễ liên kết với `pre-...` hơn.

**Comment by cuongvng** [comment]
@rootonchair đúng rồi phải là `pre-trained`, từ này đi kèm với tham số các tầng đã được huấn luyện. 
@lkhphuc em hiểu `tiền huấn luyện` sang `trước khi huấn luyện` :v

**Comment by duythanhvn** [comment]
> Mình thấy `tiền huấn luyện` cũng được, không chuẩn như `đã qua huấn luyện` nhưng vẫn đủ hiểu (theo mình) và dễ liên kết với `pre-...` hơn.

Anh đồng tình với ông giáo @lkhphuc

**Comment by cuongvng** [comment]
nếu mọi người thấy ổn thì em sẽ sửa lại. Phase 2 của fine-tuning mọi người chú ý revise lại text cho khớp với hình là được.
